plugins/dmxusb/src/libftdi-interface.cpp

```diff
@@ -242,6 +242,10 @@ bool LibFTDIInterface::open()
     }
     else
     {
+        if (ftdi_set_latency_timer(&m_handle, 1)) {
```

Comment:
This block...just no.
Changing the main FTDI interface class is VERY risky. I experimented it at my expenses.
It may work in some case and it may break some other.
Please move this in your subclass of introduce a new interface API.

Reply:
I'm ok with the fact that it should be optional.

The problem is that this command is specific with libftdi, and is clearly a hack needed for some devices. As a hack, it should be isolated, but this isolation shouldn't be hacky.

I have no clue on how QtSerialInterface or ftd2xx libraries would work with or without tuning, but I'm pretty sure that the answer won't be the same for all interfaces.

So I suggest this :
- add a new API called "set_low_latency(boolean)" that will enable wathever it needs to let the device be more reactive, as a virtual method
- add a common implementation in DmxInterface that will do nothing
- move my code in a reimplementation method in LibFTDIInterface class
- call this API in my plugin

This will let some kind of placeholder in others implementations if they need some kind of modifications too, while not disturbing anything else for others plugins.

Reply:
Let say it's yes. :-)